src/config.json

```diff
@@ -1,8 +1,14 @@
 {
-  version:  "0.1",
-  backend: {
-    url: "http://sws-istio-system.127.0.0.1.nip.io/api",
-    user: "jdoe",
-    password: "password"
+  "version":  "0.1",
+  "backend": {
+    "url": "http://sws-istio-system.127.0.0.1.nip.io/api",
+    "user": "jdoe",
+    "password": "password",
+    "endpoints":{
```

Comment:
I'm not for having everything in config. Url really makes sense because we may want to switch it depending on the environment. But IMO endpoints are not configuration. When reading the code it could be sometimes annoying to have to read several different files just in order to understand which endpoint is being used. Plus, it adds extra regexp processing that doesn't seem necessary to me. For that I'd prefer a more straight approach.

I don't know what the others think about it.

Reply:
It's odd to see this here, as well, but I'm not sure what are the best practices about how to setup this.
Final architecture will be a pod serving the api and sharing the static content, so I guess we should address that first, and perhaps second is to work on helpers setup for development mode.

Reply:
What about having the routes defined in Api.tsx (btw: should be .ts) instead of config.json? we would just call:
`api.getServices(namespace).then(...)`

Simple and efficient, no?

Reply:
works for me